### PR TITLE
Creating dataset

### PR DESCRIPTION
**Pull request by EmmaRenauld**
Modification of the file organization description

Creation of a script to reorganize Tractoflow results.

Based on our work at meeting 25/02/2020

### CODE REVIEW

File: scripts_bash/1_data_creation/organize_from_tractoflow.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# checked!
+
+################################################################################
+# This script is used to precess the original HCP data and organize it into the
```
Comment (ppoulin91) [inline scripts_bash/1_data_creation/organize_from_tractoflow.sh]: The docstring should be re-written, this is not about HCP data anymore, and there are no args provided

File: scripts_bash/1_data_creation/organize_from_tractoflow.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# checked!
```
Comment (ppoulin91) [inline scripts_bash/1_data_creation/organize_from_tractoflow.sh]: I think the file should be renamed to `organize_from_tractoflow_template.sh`, to make it clear that it should be modified before running

File: scripts_bash/1_data_creation/organize_from_tractoflow.sh [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+# Cleaning path name
+database_folder=$(realpath $database_folder)
+
+# Checking if inputs exist
+if [ ! -d $database_folder ]; then
+  echo "Invalid database_folder argument!"
+  exit
+fi
+if [ ! -d $database_folder/preprocessed ]; then
```
Comment (ppoulin91) [inline scripts_bash/1_data_creation/organize_from_tractoflow.sh]: Should this be `tractoflow_results`?

File: scripts_bash/1_data_creation/organize_from_tractoflow.sh [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+  # anat:
+  ln -s $preprocessed_folder/$subjid/Register_T1/${subjid}__t1_warped.nii.gz $subj_folder/anat/{subject_id}_t1.nii.gz
+  ln -s $preprocessed_folder/$subjid/Segment_Tissues/${subjid}__map_wm.nii.gz $subj_folder/anat/{subject_id}_wm_map.nii.gz
+
+  # masks:
+  ln -s $preprocessed_folder/$subjid/Segment_Tissues/${subjid}__mask_wm.nii.gz $subj_folder/masks/{subject_id}_wm.nii.gz
+
+  # Bundles:
+  echo "We organized tractoflow. You could now run and reorganize Recobundles if you wish"
```
Comment (ppoulin91) [inline scripts_bash/1_data_creation/organize_from_tractoflow.sh]: This should be made clearer, no?

File: scripts_bash/1_data_creation/preprocess_bundles.sh [hunk 5]
```diff
@@ -0,0 +1,79 @@
+echo "Processing bundles..."
```
Comment (ppoulin91) [inline scripts_bash/1_data_creation/preprocess_bundles.sh]: This file should not be included now, we should just make a clean one from scratch

File: scripts_bash/1_data_creation/organize_from_tractoflow.sh (hunk 1 above)
Comment (EmmaRenauld) [reply scripts_bash/1_data_creation/organize_from_tractoflow.sh]: Oups. Did not re-read this. Done.

File: scripts_bash/1_data_creation/organize_from_tractoflow.sh (hunk 2 above)
Comment (EmmaRenauld) [reply scripts_bash/1_data_creation/organize_from_tractoflow.sh]: Ok. So far, the other ones were named example_... so I will do that here.

File: scripts_bash/1_data_creation/organize_from_tractoflow.sh (hunk 4 above)
Comment (EmmaRenauld) [reply scripts_bash/1_data_creation/organize_from_tractoflow.sh]: Better now?

File: scripts_bash/1_data_creation/preprocess_bundles.sh (hunk 5 above)
Comment (EmmaRenauld) [reply scripts_bash/1_data_creation/preprocess_bundles.sh]: This is just to save the work. It is not cleaned. (There is not #checked! mark over the function).

**Pull request merged**